src/wam2layers/tracking/backtrack.py

```diff
@@ -10,6 +10,66 @@
 from wam2layers.utils.profiling import ProgressTracker
 
 
+def reduce_domain(da, config):
+    assert np.min(da.longitude.values) >= 0, "This approach assumes the raw/default longitudes run from 0-360."
+
+    # get the config domain bounds specified by the user
+    bounds = config.reduced_domain_bounds  # [south_lat, north_lat, left_lon, right_lon]
+    south_lat, north_lat = bounds[0], bounds[1]
+    left_lon, right_lon = bounds[2], bounds[3]
+
+    # check if need to map longitude to -180 to 180 for better definition of the domain
+    if left_lon > right_lon:
+        da.coords["longitude"] = (da.coords["longitude"] + 180) % 360 - 180
+        da = da.sortby(da.longitude)
+        da = da.transpose(..., "latitude", "longitude")
```

Comment:
Do you remember why that transpose was needed?

Reply:
Yes, because the sort by moves the order of latitude and longitude around in the way the array is stored, and later in the WAM code these arrays are called via slicing (e.g. line 165) so latitude and longitude must stay in the consistent order.